eng/PatchConfig.props

```diff
@@ -13,6 +13,7 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(VersionPrefix)' == '2.1.23' ">
     <PackagesInPatch>
+      Microsoft.Extensions.Caching.Memory;
```

Comment:
We are editing the `Caching.Abstractions` library as well. Do we need to add it to this list?